WORKSPACE

```diff
@@ -28,32 +28,34 @@ node_repositories()
 
 http_archive(
     name = "io_bazel_rules_closure",
-    sha256 = "59498e75805ad8767625729b433b9409f80d0ab985068d513f880fc1928eb39f",
-    strip_prefix = "rules_closure-0.3.0",
-    url = "http://bazel-mirror.storage.googleapis.com/github.com/bazelbuild/rules_closure/archive/0.3.0.tar.gz",
+    sha256 = "b29a8bc2cb10513c864cb1084d6f38613ef14a143797cea0af0f91cd385f5e8c",
```

Comment:
Fixes this error:

```
ERROR: /tmpdir/external/io_bazel_rules_closure/closure/compiler/closure_js_binary.bzl:34:21: name 'set' is not defined
```

0.8.0 is the latest release of pubref/rules_closure as of this writing